docs/docs.syscalls.js

```diff
@@ -29,5 +29,15 @@ export default {
         },
       ]);
     },
-  },
+    'config/stripe/key': function (_, payload) {
+      return Promise.resolve (
+        'pk_test_5W1GpRySJaGqo11L07q0eg1Q'
+      );
+    },
+    'config/vml/base': function (_, payload) {
+      return Promise.resolve (
+        'local.adoptapet.com:3000'
```

Comment:
@aap-tim I would recommend making this `ra-api.adoptapet.com` so that it works over on periodic.aapdev.org

Reply:
Changed made as requested.